tests/unit/views/notifications/project_crew_notification.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Project Crew Notification
+  As a user,
+  I want to be notified project crew member updates.
+
+  Scenario Outline: Twoflower is added to a project
+    Given Rincewind and Twoflower are project crew in the project Expo 2010
+    When Vetinari adds twoflower with role <role>
+    Then <user> gets notified <notification_string> about addition.
+
+    Examples:
+      | user      | role            | notification_string                                                          |
+      | Twoflower | editor          | Havelock Vetinari made you an editor of Ankh-Morpork 2010                    |
+      | Rincewind | editor          | Havelock Vetinari made Twoflower an editor of Ankh-Morpork 2010              |
+      | Vetinari  | editor          | Havelock Vetinari made Twoflower an editor of Ankh-Morpork 2010              |
+      | Twoflower | promoter        | Havelock Vetinari made you a promoter of Ankh-Morpork 2010                   |
+      | Rincewind | promoter        | Havelock Vetinari made Twoflower a promoter of Ankh-Morpork 2010             |
+      | Vetinari  | promoter        | Havelock Vetinari made Twoflower a promoter of Ankh-Morpork 2010             |
+      | Twoflower | editor,promoter | Havelock Vetinari made you an editor and promoter of Ankh-Morpork 2010       |
+      | Rincewind | editor,promoter | Havelock Vetinari made Twoflower an editor and promoter of Ankh-Morpork 2010 |
+      | Vetinari  | editor,promoter | Havelock Vetinari made Twoflower an editor and promoter of Ankh-Morpork 2010 |
+      | Twoflower | crew            | Havelock Vetinari added you to the crew of Ankh-Morpork 2010                 |
+      | Rincewind | crew            | Havelock Vetinari added Twoflower to the crew of Ankh-Morpork 2010           |
+      | Vetinari  | crew            | Havelock Vetinari added Twoflower to the crew of Ankh-Morpork 2010           |
+
+  Scenario Outline: Vetinari has invited Twoflower
+    Given Rincewind and Twoflower are project crew in the project Expo 2010
+    When Vetinari invites Twoflower to the project with a role <role>
```

Comment:
If Twoflower is already a crew member, how can Twoflower be invited now to be a crew member? The setup is wrong. Replace fixtures as above.